src/lib/token_list_gen.ts

```diff
@@ -114,20 +122,20 @@ export const generateTokenList = async (
     if (logoUris[i]) {
       arbTokenInfo = { ...{ logoURI: logoUris[i] }, ...arbTokenInfo };
     } else {
-      console.log('no logo uri for ',token.l1TokenAddr, symbol);
+      console.log('no logo uri for ',token.token.l1TokenAddr, symbol);
       
     }
 
     return arbTokenInfo;
   }).filter((tokenInfo: ArbTokenInfo)=>{
-    return tokenInfo.extensions && tokenInfo.extensions.bridgeInfo[l1Network.chainID].originBridgeAddress !== "0x0000000000000000000000000000000000000001" 
+    return tokenInfo.extensions && tokenInfo.extensions.bridgeInfo[l1.network.chainID].originBridgeAddress !== "0x0000000000000000000000000000000000000001" 
```

Comment:
we can use the DISABLED_GATEWAY constant from arb-ts here

Reply:
updated